resources/views/api/emails/voucher_history_email.blade.php

```diff
@@ -2,14 +2,18 @@
 
 @section('content')
 [xXx]
-    <p>Hi {{ $user }},</p>
+    <div role="main">
+        <h1 style="text-align: center;">Voucher payment history request</h1>
 
-    <p>You've requested a record of {{ $trader }}'s voucher payment history, which is attached to this email.</p>
-    <!-- Is this the right email> -->
-    <p>If you have any problems with opening or downloading the file attached, please <a href="mailto:arc@neontribe.co.uk">email arc@neontribe.co.uk</a>.</p>
+        <p>Hi {{ $user }},</p>
 
-    <p>Thanks,<br>
-    Rose Vouchers<br>
-    <a href="http://www.alexandrarose.org.uk/">www.alexandrarose.org.uk</a></p>
+        <p>You've requested a record of {{ $trader }}'s voucher payment history, which is attached to this email.</p>
+
+        <p>If you have any problems with opening or downloading the file attached, please <a href="mailto:arc@neontribe.co.uk">email arc@neontribe.co.uk</a>.</p>
+
+        <p>Thanks,<br>
```

Comment:
Could move this and the main at the top to another tpl so that this one is Only the content of the mail.

BUT - as this is our only mail at the moment - fine. There is no other mail to share components with.